frontend/src/components/resourceMap/useQueryParamsState.tsx

```diff
@@ -27,7 +27,7 @@ export function useQueryParamsState<T extends string | undefined>(
     const searchParams = new URLSearchParams(search);
     const paramValue = searchParams.get(param);
 
-    return paramValue !== null ? (decodeURIComponent(paramValue) as T) : initialState;
+    return paramValue !== null ? (decodeURIComponent(paramValue) as T) : undefined;
```

Comment:
Are the changes in this file intentional? I don't see any commit them for them.

Reply:
Yes, initial state wasn't initialized properly, moved it into a separate commit

Reply:
Cool, thanks.